### PR TITLE
Docker compose fix

### PR DESCRIPTION
**Pull request by goru97**
Fixed CQL version mismatch.

### CODE REVIEW

**Comment by goru97** [comment]
@shintasmith Take a look!

File: contrib/blueflood-docker/docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 blueflood:
-  build: .
+  image: goru97/blueflood
```
Comment (usnavi) [inline contrib/blueflood-docker/docker-compose.yml]: Should we have user names in our example?

Comment (goru97) [inline contrib/blueflood-docker/docker-compose.yml]: @usnavi @shintasmith had the same concern in https://github.com/rackerlabs/blueflood/pull/696. I reached out to #rackspace #respose channels on IRC, perused through one wiki and also e-mailed dev and devops list but nobody got the credentials for rackerlabs docker-hub account. usernames are part of the docker images if they are hosted on your docker hub account. They are not required if they are hosted by docker official account e.g. cassandra, elasticsearch.

Comment (usnavi) [inline contrib/blueflood-docker/docker-compose.yml]: OK, let's discuss more when you return.  Have a good break!

**Comment by usnavi** [comment]
Included in #696

**Pull request closed without merging**